homework.js

```diff
@@ -0,0 +1,118 @@
+//1 
+let arr = [1, 2, 3, 4, 5];
+function sum(arr) {
+    var s = 0;
+    for (i = 0; i < arr.length; i++) {
+       s += arr[i];
+    }
+    return s
+  }
+  
+  //2
+let numStr = [1, 2, 3, 4, 5];
+
+function sumEvens(numStr){
+  let sum = 0;
+  for (let i = 0; i < numStr.length; i++) {
+    if (numStr[i] % 2 === 0) {
+    sum = sum + numStr[i];
+    }
+  }
+  return sum;
+}
+ 
+console.log(sumEvens(numStr));
+ 
+//3 
+let arr1 = [8, 2, 2, 4]; 
+function theAverage1(arr1){
+var total = 0;
+for(var i = 0; i < arr1.length; i++) {
+    total += arr1[i];
+var avg = total / arr1.length;
+}
+return avg }
+
+//4 
+function reverseString(str) {
+    var newString = "";
+    for (var i = str.length - 1; i >= 0; i--) {
+        newString += str[i];
+    }
+    return newString;
+}
+reverseString('caterpillar');
+
+//5
+const elements1 = ['test1', 'test2', 'test3'];
+function addDashes(elements1){
+console.log(elements1.join('-'));
+}
+//6
+function countUpAndDown(i)
```

Comment:
" i " is a bad argument name, the function doesn't work because for redeclaring issue, should return a string not a console.log.